### PR TITLE
app-text/asciidoc-8.6.10_p20181026: new snapshot

### PR DESCRIPTION
**Pull request by marcecj**
Add a new snapshot ebuild based on the latest commit of Asciidoc-Py3.

Note that there are no releases yet (except for the old, Python 2 ones), but it does, in fact, pass its test suite for both CPython 3.6 and PyPy3 7.1.0.

Closes: https://bugs.gentoo.org/661662
Package-Manager: Portage-2.3.62, Repoman-2.3.11
Signed-off-by: Marc Joliet <marcec@gmx.de>

### CODE REVIEW

**Comment by marcecj** [comment]
Sorry for the late additional commit, I was perusing my bug list and was reminded that I should add that dependency.

**Comment by marcecj** [comment]
Ugh, I noticed a typo in the version number.  I'll stop revising this now.

File: app-text/asciidoc/asciidoc-8.6.10_p20181016.ebuild [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	graphviz? ( media-gfx/graphviz )
+	highlight? (
+		|| (
+			dev-util/source-highlight
+			dev-python/pygments[${PYTHON_USEDEP}]
+			app-text/highlight
+		)
+	)"
+DEPEND="
+	test? (
```
Comment (mgorny) [inline app-text/asciidoc/asciidoc-8.6.10_p20181016.ebuild]: Are you sure other RDEPs aren't needed for testing?

**Comment by marcecj** [comment]
Thanks!

**Pull request closed without merging**